# HG changeset patch
# User alta88@fixall.com
# Date 1711624612 -3600
# Parent  5908c4eb4bba54123c3802dd36c7eae1c72f5a04
Fix error when displaying feed message in a non feed account folder.

Authored by Alta88 and contributed to the Betterbird Project.
See https://github.com/Betterbird/thunderbird-patches/blob/main/LICENSE for license details.
All uses require attribution to the Author.

diff --git a/mailnews/extensions/newsblog/newsblogOverlay.js b/mailnews/extensions/newsblog/newsblogOverlay.js
--- a/mailnews/extensions/newsblog/newsblogOverlay.js
+++ b/mailnews/extensions/newsblog/newsblogOverlay.js
@@ -133,8 +133,11 @@ var FeedMessageHandler = {
         // latter, a show summary checkbox toggle in Subscribe dialog will set
         // one on the path to bliss.
         const folder = aMsgHdr.folder;
         showSummary = true;
+        if (!this.FeedAccountTypes.includes(folder.server.type)) {
+          break;
+        }
         const ds = FeedUtils.getSubscriptionsDS(folder.server);
         for (const sub of ds.data) {
           if (sub.destFolder == folder.URI) {
             showSummary = sub.quickMode;
